terraform/statcan.alpha.canada.ca.tf

```diff
@@ -13,7 +13,27 @@ resource "aws_route53_record" "statistique-alpha-canada-ca-CNAME" {
     name    = "statistique.alpha.canada.ca"
     type    = "CNAME"
     records = [
-        "www.prod.cloud.statcan.ca"
+        "www-alpha.prod.cloud.statcan.ca"
+    ]
+    ttl     = "300"
+}
+
+resource "aws_route53_record" "energy-information-alpha-canada-ca-CNAME" {
+    zone_id = aws_route53_zone.alpha-canada-ca-public.zone_id
+    name    = "energy-information.alpha.canada.ca"
```

Comment:
They're going to have to pull this to a subdomain of statistics.. ie: energy-information.statistics.alpha.canada.ca

Root level domains are meant to be "protected" to avoid turf wars. I'm open to challenge though. Have the fellas take a look and let me know.

Reply:
We already have the energy-information.canada.ca since it's a collaboration between us and ECCC + Canada Energy Regulator. I definitely understand the concern though, but I believe there should probably be exceptions for collaborative projects.

Reply:
Works for me.